kasa/cli.py

```diff
@@ -10,11 +9,6 @@
 
 from kasa import SmartPlug  # noqa: E402; noqa: E402
 
-if sys.version_info < (3, 6):
-    print("To use this script you need Python 3.6 or newer! got %s" % sys.version_info)
```

Comment:
This was here for people that use a `pip` version that doesn't handle `python_requires` yet. But probably it's safe to remove.

Reply:
Yes indeed, but those were very, very old installations :-)